src/ansys/dpf/post/simulation.py

```diff
@@ -102,9 +102,12 @@ def results(self) -> List[str]:
         >>> print(simulation.results) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
         [...]
         """
-        return [
-            str(result) for result in self._model.metadata.result_info.available_results
-        ]
+        return self._model.metadata.result_info.available_results
+
+    @property
```

Comment:
@PProfizi could you pease point me to the usage of the mesh info ?

Reply:
@cbellot000 the MeshInfo is not yet available